plugins/localization-connector/yarn.lock

```diff
@@ -0,0 +1,12361 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
what do you think of using `npm` for consistency with the rest of the packages here

Reply:
No problemo. I just find it very interesting that for instance, when having a create-react-app build problem, the react recommendations is to try having yarn instead of npm 😅